Utils/CityGML2Stripper/CityGML2Stripper.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import lxml.etree as ET
+
+def ParseCommandLine():
+    # arg parse
+    descr = '''A small utility that strips a CityGML 2.0 (XML) 
+            files and serialize the result in a new CityGML (XML) 
```

Comment:
Shouldn't the description be  
"A small utility that strips a CityGML 2.0 (XML) file [NO trailing S]  and serializeS [with a trailing S this time] the result in a new..." ?

Reply:
Yep, done